### PR TITLE
hyp + kane ai release notes

### PR DESCRIPTION
**Pull request by amanchopra1905**

### CODE REVIEW

File: docs/kaneai-release-notes-0-0-2.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+KaneAI now fully supports testing for:
+
+- Iframe elements 🖼️
+- Shadow DOM elements 🕶️
+- Canvas elements 🎨
+
+## 2. Iframe-Shadow-Canvas Support in Code Export
+We've gone the extra mile to bring full Iframe, Shadow DOM, and Canvas support in code exports 🖥️. Now, KaneAI handles these intricate UI elements with precision during script generation, making your automation journey even smoother. Whether dealing with complex nested iframes or shadow-rooted elements, KaneAI has you covered!
+
+## 3. Linux Image Fix in Test Management System (TMS) 🐧
```
Comment (Private1647) [inline docs/kaneai-release-notes-0-0-2.md:53]: This is not required. Please remove.

File: docs/kaneai-release-notes-0-0-2.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+          "@type": "ListItem",
+          "position": 3,
+          "name": "Version",
+          "item": "https://www.lambdatest.com/support/docs/kaneai-release-notes-0-0-2/"
+        }]
+      })
+    }}
+></script>
+## 1. Iframe, Shadow DOM, and Canvas Support
+KaneAI now fully supports testing for:
```
Comment (Private1647) [inline docs/kaneai-release-notes-0-0-2.md:44]: Club #1 & #2 together and add:

KaneAI now fully supports testing for:
- iframe elements 🖼️
- Shadow DOM elements 🕶️
- Canvas elements 🎨

We've gone the extra mile to bring full iframe, Shadow DOM, and Canvas support in code exports 🖥️. Now, KaneAI handles these intricate UI elements with precision during script generation, making your automation journey even smoother. Whether dealing with complex nested iframes or shadow-rooted elements, KaneAI has you covered!

File: docs/kaneai-release-notes-0-0-2.md [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+## 4. Auto-Scroll for Manual Interaction 🎯
+No more searching for the latest entries after clicking! KaneAI now automatically scrolls to the latest interaction in manual testing sessions, saving time and making testing more efficient.
+
+## 5. View Issue Button on Error Only 🚨
+To enhance UI clarity, the "View Issue" button will only appear if there's an error within the viewport. This helps testers focus on relevant issues without unnecessary clutter.
+
+## 6. Screenshot Modal Fix 📸
+We’ve fixed an issue where screenshots would appear outside the modal on both KaneAI and TMS. Screenshots will now stay neatly within their designated area for better viewing.
+
+## 7. URL Update with Objective Changes 🌐
```
Comment (Private1647) [inline docs/kaneai-release-notes-0-0-2.md:65]: Add a tag for "Test Planner"

File: docs/kaneai-release-notes-0-0-2.md [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+## 3. Linux Image Fix in Test Management System (TMS) 🐧
+We've resolved a bug impacting the Linux image within the Test Management System (TMS), ensuring a smooth and error-free experience for users on this platform.
+
+## 4. Auto-Scroll for Manual Interaction 🎯
+No more searching for the latest entries after clicking! KaneAI now automatically scrolls to the latest interaction in manual testing sessions, saving time and making testing more efficient.
+
+## 5. View Issue Button on Error Only 🚨
+To enhance UI clarity, the "View Issue" button will only appear if there's an error within the viewport. This helps testers focus on relevant issues without unnecessary clutter.
+
+## 6. Screenshot Modal Fix 📸
```
Comment (Private1647) [inline docs/kaneai-release-notes-0-0-2.md:62]: Add tag for "Test Manager" & "Web Agent"

File: docs/kaneai-release-notes-0-0-2.md [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+## 2. Iframe-Shadow-Canvas Support in Code Export
+We've gone the extra mile to bring full Iframe, Shadow DOM, and Canvas support in code exports 🖥️. Now, KaneAI handles these intricate UI elements with precision during script generation, making your automation journey even smoother. Whether dealing with complex nested iframes or shadow-rooted elements, KaneAI has you covered!
+
+## 3. Linux Image Fix in Test Management System (TMS) 🐧
+We've resolved a bug impacting the Linux image within the Test Management System (TMS), ensuring a smooth and error-free experience for users on this platform.
+
+## 4. Auto-Scroll for Manual Interaction 🎯
+No more searching for the latest entries after clicking! KaneAI now automatically scrolls to the latest interaction in manual testing sessions, saving time and making testing more efficient.
+
+## 5. View Issue Button on Error Only 🚨
```
Comment (Private1647) [inline docs/kaneai-release-notes-0-0-2.md:59]: Ignore remove this please.

File: docs/kaneai-release-notes-0-0-2.md [hunk 6]
```diff
@@ -0,0 +1,100 @@
...
+- Shadow DOM elements 🕶️
+- Canvas elements 🎨
+
+## 2. Iframe-Shadow-Canvas Support in Code Export
+We've gone the extra mile to bring full Iframe, Shadow DOM, and Canvas support in code exports 🖥️. Now, KaneAI handles these intricate UI elements with precision during script generation, making your automation journey even smoother. Whether dealing with complex nested iframes or shadow-rooted elements, KaneAI has you covered!
+
+## 3. Linux Image Fix in Test Management System (TMS) 🐧
+We've resolved a bug impacting the Linux image within the Test Management System (TMS), ensuring a smooth and error-free experience for users on this platform.
+
+## 4. Auto-Scroll for Manual Interaction 🎯
```
Comment (Private1647) [inline docs/kaneai-release-notes-0-0-2.md:56]: Add tag for "Web Agent"

File: docs/kaneai-release-notes-0-0-2.md [hunk 7]
```diff
@@ -0,0 +1,100 @@
...
+    }}
+></script>
+## 1. Iframe, Shadow DOM, and Canvas Support
+KaneAI now fully supports testing for:
+
+- Iframe elements 🖼️
+- Shadow DOM elements 🕶️
+- Canvas elements 🎨
+
+## 2. Iframe-Shadow-Canvas Support in Code Export
```
Comment (Private1647) [inline docs/kaneai-release-notes-0-0-2.md:50]: Delete and club with the one above.

File: docs/kaneai-release-notes-0-0-2.md (hunk 2 above)
Comment (Private1647) [reply docs/kaneai-release-notes-0-0-2.md:44]: Add tag for web agent

File: docs/kaneai-release-notes-0-0-2.md [hunk 8]
```diff
@@ -0,0 +1,100 @@
...
+## 5. View Issue Button on Error Only 🚨
+To enhance UI clarity, the "View Issue" button will only appear if there's an error within the viewport. This helps testers focus on relevant issues without unnecessary clutter.
+
+## 6. Screenshot Modal Fix 📸
+We’ve fixed an issue where screenshots would appear outside the modal on both KaneAI and TMS. Screenshots will now stay neatly within their designated area for better viewing.
+
+## 7. URL Update with Objective Changes 🌐
+Whenever you update an objective, the URL will now update along with any other pre-filled fields. This improvement ensures that your URLs stay in sync with your updates for better tracking and organization.
+
+## 8. Prevention of Empty API Calls 🛑
```
Comment (Private1647) [inline docs/kaneai-release-notes-0-0-2.md:68]: Remove please

File: docs/kaneai-release-notes-0-0-2.md [hunk 9]
```diff
@@ -0,0 +1,100 @@
...
+We've added logic to prevent empty planner API calls on the main objective page. This optimization reduces unnecessary network activity and speeds up the experience.
+
+## 9. Label Overlap Fix in TMS 🏷️
```
Comment (Private1647) [inline docs/kaneai-release-notes-0-0-2.md:71]: Add "Test manager" Tag

File: docs/kaneai-release-notes-0-0-2.md [hunk 10]
```diff
@@ -0,0 +1,100 @@
...
+## 12. Excluding KaneAI Test Cases from Test Plan & Build Creation ⚠️
+For better test management, we’ve restricted the creation of test plans and builds to exclude KaneAI Test Cases, ensuring a more streamlined process.
+
+## 13. Manual Interaction Enhancements 🖱️
+We've refined manual interactions to make Click actions faster and more precise, optimizing the overall experience for testers.
+
+## 14. Stable Template Support 📑
+Our templates have been stabilized for more reliable performance during test creation and execution.
+
+## 15. Appium Test Generation (Frontend Support) 📱
```
Comment (Private1647) [inline docs/kaneai-release-notes-0-0-2.md:89]: Remove please. We have not yet release Mobile support in beta

File: docs/kaneai-release-notes-0-0-2.md [hunk 11]
```diff
@@ -0,0 +1,100 @@
...
+While not live yet, we’ve added support for Appium test generation from the frontend, getting ready for future mobile testing capabilities.
+
+## 16. Meta Tag Updates for KaneAI 🏷️
```
Comment (Private1647) [inline docs/kaneai-release-notes-0-0-2.md:92]: Remove please

File: docs/kaneai-release-notes-0-0-2.md [hunk 12]
```diff
@@ -0,0 +1,100 @@
...
+Meta tags have been updated for KaneAI, improving SEO and overall discoverability.
+
+## 17. Enhancements in Visual and Textual Query for Code Export 🔍
```
Comment (Private1647) [inline docs/kaneai-release-notes-0-0-2.md:95]: Change "Code Export" to "Code generation" in title as well as in body for this one.

Add a tag for "Web Agent"

File: docs/kaneai-release-notes-0-0-2.md [hunk 13]
```diff
@@ -0,0 +1,100 @@
...
+We've enhanced the visual and textual query capabilities in code exports, making your scripts even more accurate when handling complex UI elements.
+
+## 18. UI Action Improvements 🎯
```
Comment (Private1647) [inline docs/kaneai-release-notes-0-0-2.md:98]: This is part of code generation above. "UIActions" is the python file generated. Please remove.

File: docs/kaneai-release-notes-0-0-2.md [hunk 14]
```diff
@@ -0,0 +1,100 @@
...
+## 11. Page Title Update 🔄
+We’ve officially rebranded all page titles from Auteur to KaneAI, making our identity consistent across all UI elements.
+
+## 12. Excluding KaneAI Test Cases from Test Plan & Build Creation ⚠️
+For better test management, we’ve restricted the creation of test plans and builds to exclude KaneAI Test Cases, ensuring a more streamlined process.
+
+## 13. Manual Interaction Enhancements 🖱️
+We've refined manual interactions to make Click actions faster and more precise, optimizing the overall experience for testers.
+
+## 14. Stable Template Support 📑
```
Comment (Private1647) [inline docs/kaneai-release-notes-0-0-2.md:86]: Add tag for "Web Agent" These templates are the examples shown when KaneAI is open. There were some issues which led to failure which have been fixed now. Please update.

File: docs/kaneai-release-notes-0-0-2.md [hunk 15]
```diff
@@ -0,0 +1,100 @@
...
+## 10. Test Case Name Display Issue Fixed 🔍
+Resolved an issue where test case names weren't appearing in certain situations, such as when the screen was zoomed in at 200%. This fix ensures that test case names are visible even in extreme zoom conditions.
+
+## 11. Page Title Update 🔄
+We’ve officially rebranded all page titles from Auteur to KaneAI, making our identity consistent across all UI elements.
+
+## 12. Excluding KaneAI Test Cases from Test Plan & Build Creation ⚠️
+For better test management, we’ve restricted the creation of test plans and builds to exclude KaneAI Test Cases, ensuring a more streamlined process.
+
+## 13. Manual Interaction Enhancements 🖱️
```
Comment (Private1647) [inline docs/kaneai-release-notes-0-0-2.md:83]: Add tag for "Manual interaction" & "Web Agent"

File: docs/kaneai-release-notes-0-0-2.md [hunk 16]
```diff
@@ -0,0 +1,100 @@
...
+## 9. Label Overlap Fix in TMS 🏷️
+Fixed the issue of labels overlapping other elements on the TMS UI. Now, we show a concise label such as "+12" when there's no space. Clicking it will open an overlay with all label details, improving readability and UI neatness.
+
+## 10. Test Case Name Display Issue Fixed 🔍
+Resolved an issue where test case names weren't appearing in certain situations, such as when the screen was zoomed in at 200%. This fix ensures that test case names are visible even in extreme zoom conditions.
+
+## 11. Page Title Update 🔄
+We’ve officially rebranded all page titles from Auteur to KaneAI, making our identity consistent across all UI elements.
+
+## 12. Excluding KaneAI Test Cases from Test Plan & Build Creation ⚠️
```
Comment (Private1647) [inline docs/kaneai-release-notes-0-0-2.md:80]: Remove please. We do not want to tell. this to people.

File: docs/kaneai-release-notes-0-0-2.md [hunk 17]
```diff
@@ -0,0 +1,100 @@
...
+## 8. Prevention of Empty API Calls 🛑
+We've added logic to prevent empty planner API calls on the main objective page. This optimization reduces unnecessary network activity and speeds up the experience.
+
+## 9. Label Overlap Fix in TMS 🏷️
+Fixed the issue of labels overlapping other elements on the TMS UI. Now, we show a concise label such as "+12" when there's no space. Clicking it will open an overlay with all label details, improving readability and UI neatness.
+
+## 10. Test Case Name Display Issue Fixed 🔍
+Resolved an issue where test case names weren't appearing in certain situations, such as when the screen was zoomed in at 200%. This fix ensures that test case names are visible even in extreme zoom conditions.
+
+## 11. Page Title Update 🔄
```
Comment (Private1647) [inline docs/kaneai-release-notes-0-0-2.md:77]: Remove please. They are not aware of what Auteur is.

**Review by Private1647 — CHANGES_REQUESTED** [review]
@amanchopra1905 - Please check comments and update.

**Review by Private1647 — APPROVED** [review]

**Review by surishubham — APPROVED** [review]

**Pull request merged**